decidim/decidim
# Remove code duplication in commands (part 9)

#### :tophat: What? Why?

This PR attempts to standardize the way the file attachments are handled within the commands. This is actually the implementation that I was speaking with @andreslucena in this thread: https://github.com/decidim/decidim/pull/12183#discussion_r1430224015

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #12183
- 
#### Testing
1. Make sure the pipeline is green, 
2. Attempt to create any new participatory space (assembly, participatory process, conference) 
3. Attach images to it and click save 
4. Change the images and click save
5. See there are no errors, and the images are properly persisted
6. Additionally, try to add an invalid file type and see the error

:hearts: Thank you!
